workbench-core/user-management/src/plugins/cognitoUserManagementPlugin.test.ts

```diff
@@ -1182,4 +1230,84 @@ describe('CognitoUserManagementPlugin tests', () => {
       await expect(plugin.deleteRole(roles[0])).rejects.toThrow(Error);
     });
   });
+
+  describe('validateUserRoles', () => {
```

Comment:
Should we test for expired access entry? E.g. set TTL to 500 ms and verify that expires behaves as expected.

Reply:
it does check for expiration under the test `Do not modify roles if user is temporarily revoked access but has expired`